packages/app-headless-cms-common/src/types/index.ts

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import ApolloClient from "apollo-client";
```

Comment:
We can use `import type ApolloClient` here so that's explicit we need this only for the type.